### DISCUSSION TITLE
重复：如何记住“无边框‘’状态？

### DISCUSSION THREAD

**Discussion by jolyonaha** [hooke007/mpv_PlayKit#152] [回收]
![image](https://user-images.githubusercontent.com/67818339/179355700-c90ebabc-630c-4ca7-afd5-eb603c4faa1a.png)
像这样，如何在关闭再次启动后仍然是无边框？

**Comment by hooke007** [comment]
选项设置里不就有吗？

**Reply by jolyonaha** [reply]
找到了，感谢（｡ò ∀ ó｡）

**Reply by jolyonaha** [reply]
> 选项设置里不就有吗？

![image](https://user-images.githubusercontent.com/67818339/179358777-e1119ed3-e78a-4c1f-be89-2ee8d31ef06d.png)
打扰了，最后再请教一下自动开启mvtools60的代码，我的这行是无效的😗

**Reply by hooke007** [reply]
善用搜索
https://github.com/hooke007/MPV_lazy/blob/3601dd1f2adafd5eeb80afe8b2996b9af36493d6/portable_config/mpv.conf#L217
